src/wp-includes/class-wp-theme.php

```diff
@@ -194,6 +194,25 @@ final class WP_Theme implements ArrayAccess {
 	 */
 	private $cache_hash;
 
+	/**
+	 * Block template folders.
+	 *
+	 * @since 6.4.0
+	 * @var array|null
```

Comment:
The `get_block_template_folders` function return type shows different return value then this. In which case we get null value?

```
* @return string[] {
*     Folder names used by block themes.
*
*     @type string $wp_template      Theme-relative directory name for block templates.
*     @type string $wp_template_part Theme-relative directory name for block template parts.
* }
```

Reply:
@mukeshpanchal27  You would never get a null for `get_block_template_folders` as we have a defaulting logic. But, variable `block_template_folders` could be null, if it is not set or when the cache is cleared.  
However I feel this is the case with other variables too and they aren't mentioning that. So I'll remove it.